README.md

```diff
@@ -1,12 +1,14 @@
-Alpakka [![scaladex-badge][]][scaladex] [![travis-badge][]][travis] [![gitter-badge][]][gitter]
+Alpakka [![scaladex-badge][]][scaladex] [![maven-central-badge][]][maven-central] [![travis-badge][]][travis] [![gitter-badge][]][gitter]
 =======
 
-[scaladex]:       https://index.scala-lang.org/akka/alpakka
-[scaladex-badge]: https://index.scala-lang.org/akka/alpakka/latest.svg
+[scaladex]:              https://index.scala-lang.org/akka/alpakka
+[scaladex-badge]:        https://index.scala-lang.org/akka/alpakka/latest.svg
+[maven-central]:         https://maven-badges.herokuapp.com/maven-central/com.lightbend.akka/akka-stream-alpakka-file_2.12
+[maven-central-badge]:   https://maven-badges.herokuapp.com/maven-central/com.lightbend.akka/akka-stream-alpakka-file_2.12/badge.svg
```

Comment:
While I agree that linking to the `http://search.maven.org` is useful, I do not like that it goes to one particular connector when clicked on.

That is why I like the scaladex badge which links to scaladex alpakka page which then has all connectors.

Reply:
I don't see a way to link to several Alpakka modules at search.maven.org.
Closest I get is http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.lightbend.akka%22%20AND%20v%3A%220.18%22 which needs to be updated manually.

Reply:
It seems that maven.org has a new search page which supports searching by partial artifact name. So this works: https://search.maven.org/beta/search?q=akka-stream-alpakka

Reply:
Updated.